spec/models/rent_spec.rb

```diff
@@ -76,78 +58,72 @@
         end
         it 'add error if d_from > d_til' do
           rent.d_til = Time.zone.now - 10.hours
           rent.d_from = rent.d_til + 5.hours
-          rent.valid?
-          rent.errors.get(:d_from).should include("måste vara i framtiden.")
+          rent.should_not be_valid
+          rent.errors.get(:d_from).should include(I18n.t('rent.validation.future'))
         end
         it 'add error if d_from < d_til' do
           rent.d_from = Time.zone.now - 10.hours
           rent.d_til = rent.d_from + 5.hours
-          rent.valid?
-          rent.errors.get(:d_from).should include("måste vara i framtiden.")
+          rent.should_not be_valid
+          rent.errors.get(:d_from).should include(I18n.t('rent.validation.future'))
         end
       end
     end
 
     describe :Overlap do
       context :when_no_councils do
-        let(:overlap) { create(:rent, :good) }
-        let(:normal) { build(:rent, :good) }
         it 'invalid when d_til is within saved rent' do
-          normal.d_from = overlap.d_from - 5.hours
-          normal.d_til = overlap.d_til - 5.hours
-          normal.should_not be_valid
+          rent_n.d_from = rent.d_from - 5.hours
+          rent_n.d_til = rent.d_til - 5.hours
+          rent_n.should_not be_valid
         end
         it 'invalid when d_from and d_til is within saved rent' do
-          normal.d_from = overlap.d_from + 5.hours
-          normal.d_til = overlap.d_til - 5.hours
-          normal.should_not be_valid
+          rent_n.d_from = overlap.d_from + 5.hours
+          rent_n.d_til = overlap.d_til - 5.hours
+          rent_n.should_not be_valid
         end
         it 'add error if d_from is within saved' do
-          normal.d_from = overlap.d_from + 5.hours
-          normal.d_til = overlap.d_til - 5.hours
-          normal.valid?.should be_falsey
-          normal.errors.get(:d_from).should include("överlappar med annan bokning")
+          rent_n.d_from = overlap.d_from + 5.hours
+          rent_n.d_til = overlap.d_til - 5.hours
+          rent_n.valid?.should be_falsey
+          rent_n.errors.get(:d_from).should include("överlappar med annan bokning")
         end
         it 'invalid when d_from is within saved rent' do
-          normal.d_from = overlap.d_til - 5.hours
-          normal.d_til = overlap.d_til + 5.hours
-          normal.should_not be_valid
+          rent_n.d_from = overlap.d_til - 5.hours
+          rent_n.d_til = overlap.d_til + 5.hours
+          rent_n.should_not be_valid
         end
         it 'do add error if d_from is within saved' do
-          normal.d_from = overlap.d_til - 5.hours
-          normal.d_til = overlap.d_til + 5.hours
-          normal.valid?
-          normal.errors.get(:d_from).should include("överlappar med annan bokning")
+          rent_n.d_from = overlap.d_til - 5.hours
+          rent_n.d_til = overlap.d_til + 5.hours
+          rent_n.valid?
+          rent_n.errors.get(:d_from).should include("överlappar med annan bokning")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.